create_config.sh

```diff
@@ -7,11 +7,8 @@ exports.config = {
   client_secret: "${SPHERE_CLIENT_SECRET}",
   project_key: "${SPHERE_PROJECT_KEY}"
 }
-EOF
-
-cat > "test-config.js" << EOF
 /* IRON.IO MQ credentials */
-exports.config = {
+exports.ironConfig = {
```

Comment:
If I start putting everything in one config object, the object can become too big. Additionally, I can also have object keys with same names.